src/lib/helpers.ts

```diff
@@ -1,28 +1,41 @@
-import fetch from 'node-fetch';
+import * as fetch from 'isomorphic-fetch';
 
-export async function request(uri: string, headers: { [header: string]: string; } = {}) {
-    const req = await fetch(uri, { compress: true, timeout: 30000, headers });
-    if (req.status !== 200) {
-        return null;
-    }
-    return await req.json();
+/**
+ * Make a http(s) request to a json api.
+ *
+ * Throws an error when a network error occured.
```

Comment:
According to the [github documentation](https://github.github.io/fetch/#error), where the module is based upon, it does.